src/radio-tile/radio-tile.scss

```diff
@@ -0,0 +1,143 @@
+  &:first-of-type {
+    border-radius: $iui-border-radius 0 0 $iui-border-radius;
+  }
+
+  // Rounded corners on last tile
+  &:last-of-type {
+    border-radius: 0 $iui-border-radius $iui-border-radius 0;
+  }
+
+  // All tiles shift 1px left so we don't have 2px thick dividers
+  &:not(:first-of-type) {
+    margin-left: -1px;
+  }
+
+  &-container {
+    display: inline-flex;
+    flex-wrap: wrap;
+    width: fit-content;
+    user-select: none;
+    // Create stacking context
+    position: relative;
+    z-index: 0;
+  }
+
+  &-input {
+    width: 0;
+    height: 0;
+    appearance: none;
+    opacity: 0;
+    position: absolute;
+    @include focus-visible {
+      + * {
+        outline: 2px solid var(--iui-color-foreground-primary);
+        outline-offset: -4px;
+      }
+    }
+
+    + *:hover {
+      z-index: 2;
+      border-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-2));
+
+      > * {
+        fill: var(--iui-icons-color-actionable);
+      }
+    }
+    // Styles checked radio tile
+    &:checked + * {
+      padding: $iui-s - 1;
+      z-index: 3;
+      border: 2px solid var(--iui-color-foreground-primary);
+
+      > * {
+        fill: var(--iui-icons-color-primary);
+      }
+
+      &::after {
+        content: '';
+        width: $iui-icons-default;
+        height: $iui-icons-default;
+        position: absolute;
+        top: $iui-s;
+        right: $iui-s;
+        background-color: var(--iui-color-foreground-primary);
+        mask: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z" /></svg>');
+      }
+    }
+
+    // Styles disabled radio tile
+    &:disabled + * {
+      cursor: not-allowed;
+      z-index: 0;
+      border-color: var(--iui-color-background-disabled);
+      background-color: var(--iui-color-background-disabled);
+
+      &::after {
+        background-color: var(--iui-icons-color-actionable-disabled);
+      }
+
+      .iui-radio-tile-icon {
+        fill: var(--iui-icons-color-actionable-disabled);
+        filter: $iui-icons-color-multicolor-disabled;
+      }
+
+      .iui-radio-tile-label {
+        color: var(--iui-text-color-muted);
+      }
+    }
+
+    // Styles disabled checked radio tile
+    &:disabled:checked + * {
+      z-index: 3;
+      border: 2px solid rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+    }
+  }
+
+  &-icon {
+    width: $iui-icons-xl;
+    height: $iui-icons-xl;
+    display: block;
+    margin: 0 auto;
+    margin-top: round($iui-baseline * 0.5);
+    margin-bottom: $iui-baseline;
+    fill: var(--iui-icons-color);
+    @media (prefers-reduced-motion: no-preference) {
+      transition: fill $iui-speed-fast ease-out;
+    }
+  }
+
+  &-label {
+    font-size: $iui-font-size;
+    font-weight: $iui-font-weight-semibold;
+    line-height: $iui-line-height;
+    text-align: center;
+    color: var(--iui-text-color);
+  }
+
+  &-description {
```

Comment:
Hmm.. A little bit inconsistent here. If we have label, then we should have sublabel.

Reply:
I've seen instances in `header`, `menu`, `tabs`, and `tile` that use `-description` in their classes. I don't see any that use `sublabel`

Reply:
ahh.. we need to review that. Cause if we go `name-description` pattern, we cannot have same in `react` because `name` is a reserved native property.
`label-sublabel` kinda makes more sense than `label-description` to me.

Reply:
I was considering the structure like this when I worked on the failed header PR because the ID # wasn't really a "description":
```scss
.name {
	.label { … }
	.sublabel { … }
}
```
I do agree that whatever we decide on we should be consistent across all of our components.

Reply:
I would say change. It is just a rename of class if we agree from now on to go label/sublabel pattern.
Should be do this rename everywhere before 1.0 then too? @FlyersPh9

Reply:
> I like `sublabel` more than `description`
> 
> Should I change `description` to `sublabel` in this PR or is out of scope?

I meant to ask if I should rename this everywhere in this PR such as in `header` , `menu` , etc.

Reply:
`iui-radio-tile-description` changed to `iui-radio-tile-sublabel`